sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2159,6 +2174,34 @@ public void setSendModules(boolean sendModules) {
     this.sendModules = sendModules;
   }
 
+  /**
+   * Whether to send check-ins for monitors (CRONS) automatically.
+   *
+   * @param enableAutomaticCheckIns true if check-ins should be sent automatically.
+   */
+  public void setEnableAutomaticCheckIns(boolean enableAutomaticCheckIns) {
+    this.enableAutomaticCheckIns = enableAutomaticCheckIns;
+  }
+
+  public void setIgnoredCheckIns(final @Nullable List<String> ignoredCheckIns) {
+    if (ignoredCheckIns == null) {
+      this.ignoredCheckIns = null;
+    } else {
+      @NotNull final List<String> filteredIgnoredCheckIns = new ArrayList<>();
+      for (String slug : ignoredCheckIns) {
+        if (!slug.isEmpty()) {
```

Comment:
should we also check for `blank` here?

Reply:
I just copied what we do for tracing origins so I'd expect this to be sufficient. Can always improve later.